governance/governance.md

```diff
@@ -0,0 +1,255 @@
+# Governance
+
+According to the Business Dictionary, governance is defined as:
+
+> _Establishment of policies, and continuous monitoring
+> of their proper implementation, by the members of the
+> governing body of an organization. It includes the
+> mechanisms required to balance the powers of the
+> members (with the associated accountability), and
+> their primary duty of enhancing the prosperity and
+> viability of the organization_
+
+In open source, governance is described in the "governance
+model" document, defined by OSSWatch[^1] as:
+
+> _A governance model describes the roles that project participants 
+> can take on and the process for decision making within the project. 
+> In addition, it describes the ground rules for participation in the 
+> project and the processes for communicating and sharing within 
+> the project team and community_
+
+Usually the governance model is a written document containing:
+
+-  project goals
+-  work, management and collaboration infrastructures definition
+-  people roles and responsibilities definitions
+-  community support mechanisms
+-  decision making process policies description
+-  contribution process policies description
+-  monitoring policies and mechanisms description
+
+
+# Govenance Standards/Models
+
+There are some initiatives describing what good corporate open
+source governance means. The most mature one is the _Good Governance
+Initiative_ by the OSPO Alliance, which  has already released the
+first version (v1) of its framework.
+
+Coming versions of this framework are expected to consider
+InnerSource as a corporate practice managed by the Open Source
+Program Office (OSPO).
+
+In the shared spirit of reusing materials, we'll avoid overlapping
+and focus here on a pure InnerSource case where no formal corporate
+management of open source exists yet.
+
+The GGI proposes 5 goals to structure 5 governance activities for
+each of them.
+
+All GGI goals for open source governance apply with minor tweaks to
+our pure InnerSource case:
+
+- Goal 1: Foster **usage** of InnerSource software
+- Goal 2: **Build trust** around InnerSource
+- Goal 3: **Spread** the (InnerSource) **culture**
+- Goal 4: **Engage** with the (InnerSource) **community**
+- Goal 5: **Make InnerSource Strategic** for the organization
+
+There will be differences in the context that will cause an
+adaptation.
+
+## Foster usage of InnerSource software
+
+InnerSource-developed software might suffer from the same kind of
+distrust open source has. Usage is the base: don't expect
+contributions without users. And the borders applying to
+InnerSource limit the audience, so this can become challenging for
+small or varied corporations.
+
+### Activities
+
+InnerSource will have a greater need for software inventories due
```

Comment:
That's just because [Inventories](https://ospo-alliance.org/ggi/activities/inventory_of_open_source_skills_and_resources/) is the first activity in GGI.